src/schema.rs

```diff
@@ -121,19 +124,19 @@ mod schema_tests {
         load(r#" // For a schema with nested self reference type as below:
             type:: { name: my_int, type: { type: my_int } }
         "#).into_iter(),
-        2 // this includes my_int type and the anonymous type that uses my_int
+        1 // this includes my_int type and the anonymous type that uses my_int
```

Comment:
These numbers have been updated but the comments seem out of date.

Confirming my understanding: the anonymous type is _not_ returned, right? Because it isn't an "exported" type (i.e. a named type another schema could import)?

Reply:
Yes. correct. I will modify the comments for this.